src/chart_types/xy_chart/rendering/utils.ts

```diff
@@ -67,6 +67,25 @@ export function isDatumFilled({ filled, initialY1 }: DataSeriesDatum) {
 export function getClippedRanges(dataset: DataSeriesDatum[], xScale: Scale, xScaleOffset: number): ClippedRanges {
   let firstNonNullX: number | null = null;
   let hasNull = false;
+
+  const completeDatasetIsNull = dataset.filter((data) => !isDatumFilled(data)).length === 0;
```

Comment:
This could be simplified a little

```suggestion
  const completeDatasetIsNull = dataset.every((datum) => isDatumFilled(datum));
```